NEWS.md

```diff
@@ -38,6 +38,21 @@
 
 ## Features and fixes
 
+* `_lgl()`, `_int()`, `_int()`, and `_dbl()` now use the same (strict) coercion
+  methods as vctrs (#904). This means that:
+  
+    * `map_chr(TRUE, identity)`, `map_chr(0L, identity)`, and
+      `map_chr(1L, identity)` now fail because converting a 
+      logical/integer/double to a character vector should require an explicit
+      coercion.
+      
+    * `map_dbl(1.5, identity)` now fails because you shouldn't silently truncate.
+      But note that `map_dbl(1, identity)` since no numeric precision is lost.
```

Comment:
```suggestion
    * `map_int(1.5, identity)` now fails because you shouldn't silently truncate.
      But note that `map_int(1, identity)` since no numeric precision is lost.
```

Reply:
"shouldn't" seems to make assumptions here. Maybe:

> now fails to protect the caller from unexpected truncations

Reply:
Oh yeah I was trying to make a general statement, i.e. "it's a bad idea to truncate". I'll reword.